docs/refs/api/rtos/callbacks.md

```diff
@@ -306,13 +195,14 @@ thing_t thing2;
 adc.attach(callback(catinthehat, &thing2));
 ```
 
-An important thing to note, is that this state is restricted to a single pointer. This means you can’t bind both an object and argument to a callback.
+This state is restricted to a single pointer. This means you can’t bind both an object and argument to a callback.
```

Comment:
Can we make this a note thing or add some sort of emphasis, I get asked this question a lot and this specific point surprises people.